src/main/java/duke/exceptions/tasks/EmptyDescriptionException.java

```diff
@@ -0,0 +1,20 @@
+package duke.exceptions.tasks;
+
+import duke.exceptions.DukeException;
+
+/**
+ * Represents an exception that is thrown when a task is initialized with an empty description.
+ * This class extends the {@link DukeException} to provide a specific error message
+ * indicating that a task description cannot be empty.
+ */
+public class EmptyDescriptionException extends DukeException {
+    private static final String errorMessage = "The description of a task cannot be empty.";
```

Comment:
Most of your other naming conventions for private static final constants are named in all caps, so it may be better for you to change this to all caps as well for consistency? (i.e. `ERROR_MESSAGE`)

Reply:
Thanks for pointing this out! I will amend accordingly :)